values/mainnet/network/space06-1.yaml

```diff
@@ -9,7 +9,7 @@ init:
   clearRestart:
     enabled: true
 
-Lotus:
+lotus:
```

Comment:
Are you sure we need to have this change?
I think we need to revert or make additional changes